applications/basic_networking_ping/python/basic_networking_ping.py

```diff
@@ -65,8 +66,8 @@ class App(Application):
     def compose(self):
         # Define the tx and rx operators, allowing the tx operator to execute 10 times
         if len(self.kwargs("network_tx")) > 0:
-            tx = BasicNetworkPingTxOp(self, CountCondition(self, NUM_MSGS), name="tx")
             basic_net_tx = BasicNetworkOpTx(self, name="basic_net_tx", **self.kwargs("network_tx"))
+            tx = BasicNetworkPingTxOp(self, CountCondition(self, NUM_MSGS), name="tx")
```

Comment:
Does order matter here? If so, why?

Reply:
I moved it while I was debugging the issue, but apparently, the order doesn't matter. 
I kept the change so the both Rx and Tx define the operators in the same order.

Reply:
Great, thanks for confirming 👍